pkg/hubble/metrics/api/api.go

```diff
@@ -85,7 +87,9 @@ func (h Handlers) ProcessFlow(flow v1.Flow) {
 	}
 }
 
-var registry = NewRegistry()
+var registry = NewRegistry(
+	logging.DefaultLogger.WithField(logfields.LogSubsys, "hubble"),
```

Comment:
nit: I know this is a cleanup PR, so marking this as optional change - but I would prefer to use `subsys=hubble-metrics` here. 

I think metrics is a big package enough package to deserve its own logging category, especially since it's a subsystem that can be enabled/disabled by the user.

Reply:
I'm happy to change this but I wasn't sure this is what we want mostly because I'm not sure about the implications with regard to structured logging (eg: one wants to filter all hubble logs).

Reply:
i guess you can still grep for `subsys=hubble` and get all the hubble related metrics including `hubble-metrics` subsystem. personally i can go either way.